Doc/docker config (#1415)
kestra-io/docs: content/docs/02.installation/02.docker.md

The above command starts Kestra with an embedded H2 database. If you want to use a persistent database backend with Postgres and more configurability, follow the [Docker Compose installation](/docs/installation/docker-compose).
::

## Configuration

### Using a configuration file

> You can adjust the Kestra configuration with a configuration file passed to the Docker container as a bind volume.
> 
> First, create a configuration file, for example in a file named `application.yaml`:
> 

Reviewer: ```suggestion
You can adjust Kestra's configuration using a file passed to the Docker container as a bind volume.

First, create a configuration file, for example, in a file named `application.yaml`:
```

```yaml
datasources:
  postgres:
    url: jdbc:postgresql://postgres:5432/kestra
    driverClassName: org.postgresql.Driver
    username: kestra
    password: k3str4
kestra:
  server:
    basic-auth:
      enabled: false
      username: "admin@kestra.io" # it must be a valid email address
      password: kestra
  repository:
    type: postgres
  storage:
    type: local
    local:
      base-path: "/app/storage"
  queue:
    type: postgres
  tasks:
    tmp-dir:
      path: /tmp/kestra-wd/tmp
          url: http://localhost:8080/"
```

Note: this configuration is taken from our official [docker-compose.yaml](https://github.com/kestra-io/kestra/blob/develop/docker-compose.yml#L33) file and uses a PostgreSQL database, you may want to retrieve it there to be sure it is accurate.

> Then change the command to mount the file inside the container and starts Kestra with the configuration file, we also adjust the Kestra command to start a standalone version as we now have a PostgreSQL database as a backend.

Reviewer: ```suggestion
Then, change the command to mount the file to the container and start Kestra with the configuration file; we also adjust the Kestra command to start a standalone version as we now have a PostgreSQL database as a backend.
```

```bash
docker run --pull=always --rm -it -p 8080:8080 --user=root \
 -v $PWD/application.yaml:/etc/config/application.yaml
 -v /var/run/docker.sock:/var/run/docker.sock \
 -v /tmp:/tmp kestra/kestra:latest-full server standalone --config /etc/config/application.yaml
```

### Using the `KESTRA_CONFIGURATION` environment variable

You can adjust the Kestra configuration with a `KESTRA_CONFIGURATION` passed to the Docker container via the `-e` options.

> This environment variable must be a valid YAML.

Reviewer: ```suggestion
This environment variable must be a valid YAML string.
```

> As working with multi-line environment variables is not easy, we recommand configuring Kestra with a configuration file.

Reviewer: ```suggestion
Managing a large configuration via a single YAML string can be tedious. To make that easier, you can leverage a configuration file.
```
